apps/webapp/src/state/send.ts

```diff
@@ -152,3 +161,17 @@ export const sendValidationErrors = (
 };
 
 export const sendSelector = (state: AllSlices) => state.send;
+
+export const getTotalGasPriceFromGasPrices = (gasPrices: GasPrices | undefined) => {
+  if (!gasPrices) return undefined;
+
+  return (
+    gasPrices.blockSpacePrice +
+    gasPrices.compactBlockSpacePrice +
+    gasPrices.verificationPrice +
+    gasPrices.executionPrice
+  );
```

Comment:
I'm summing up the different dimensions of the gas price to get the total. I'm not 100% certain that this logic is correct — this is where I'm bumping against the limits of my crypto knowledge — so reviewers, please let me know if I've got this right.

Reply:
it looks like the final step of calculating a total might be the location where the denominator is expected to be considered

https://github.com/penumbra-zone/penumbra/blob/main/crates/core/component/fee/src/gas.rs#L65-L74